### ISSUE TITLE
Dead dependency and security vulnerability 

### ISSUE THREAD

**Issue by drifterz28** [elwin013/parcel-plugin-static-files-copy#45]
the package https://github.com/douzi8/file-match depends on package https://github.com/douzi8/utils-extend both are from the same developer and utils-extend has a vulnerability. Both seem to be dead. Is there a plan to update this package as it seems to still be active?

https://github.com/douzi8/utils-extend/issues/2

**Comment by msanatan** [comment]
Just to add to this, when I run `npm audit` in my project this is the output I get:

```plaintext
                       === npm audit security report ===                     
                                                                                
┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ utils-extend                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-plugin-static-files-copy [dev]                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-plugin-static-files-copy > file-system > file-match > │
│               │ utils-extend                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1502                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ utils-extend                                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ No patch available                                           │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ parcel-plugin-static-files-copy [dev]                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ parcel-plugin-static-files-copy > file-system > utils-extend │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1502                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Peeking at the source code (which is simple and easy to follow, good stuff!), it seems that the `file-system` and `path` modules are packages that are use the same API as the native NPM modules. I'm working on a game jam now but maybe using the native modules might be the better remedy?

**Comment by ailwynm** [comment]
Is there any particular reason it's using `file-system` and not the native one? Seems like a simple enough change to remove the sour dependencies.

**Comment by elwin013** [comment]
@drifterz28  @msanatan @ailwynm 

Should be fixed with 2.3.2 version (already published to npm). Thanks for reporting :-)

**Comment by msanatan** [comment]
Looks great, thanks @elwin013
